test/sync-tag_test.js

```diff
@@ -39,7 +39,7 @@ util.execSync = command => {
     return "";
 };
 
-ruleTester.run("require-static-url", rule, {
+ruleTester.run("sync-tag", rule, {
```

Comment:
**note:** Pretty sure this name was wrong. Sure was confusing me when I was trying to find if my tests worked.